make/modules/java.desktop/lib/Awt2dLibraries.gmk

```diff
@@ -570,14 +560,16 @@ endif
 
 TARGETS += $(BUILD_LIBFONTMANAGER)
 
+################################################################################
+## Build libjawt
 ################################################################################
 
-ifeq ($(call isTargetOs, windows), true)
-
-  LIBJAWT_CFLAGS := -EHsc -DUNICODE -D_UNICODE
+LIBJAWT_EXTRA_HEADER_DIRS := \
+    include \
+    #
```

Comment:
A syntax question - what does the trailing # do / mean here ?
Superficially I'd expect this to expand to "include #"
which means there'd be a # embedded when you append the other folders below ..

Reply:
The `#` marks a comment, so will basically be ignored. We use this construct to define easy to read lists where every element can have a trailing `\`, including the last. This makes it easier to add or remove items in the list without causing diffs in unrelated lines. In this case the list has only one element, so I would suggest just using a single line assignment. OTOH, it seems Magnus chose to use this construct for uniformity with all other HEADER and SRC lists.

Reply:
`#` is the line comment marker in makefiles.

What you see here is a common pattern in our makefiles, typically used in multi-line lists. The trailing # is a trick to avoid having a special treatment on the last line (without a `\`), which is all too easy to forget, and very hard to debug.

This particular list is very short, so `LIBJAWT_EXTRA_HEADER_DIRS := include` would have yielded the same result, and I would normally have chosen that syntax, but in this case the list will be conditionally appended to just below, so in a conceptual sense the list is much longer, but the limited syntax in makefiles does not allow for us to express this in a better way.